go.mod

```diff
@@ -4,6 +4,7 @@ go 1.16
 
 require (
 	github.com/stretchr/testify v1.7.0
-	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
-	golang.org/x/sys v0.0.0-20221010170243-090e33056c14
+	golang.org/x/net v0.7.0
```

Comment:
The `golang.org/x/net` requires `Go 1.17` in its `go.mod`, maybe we can upgrade our Go version to the `Go 1.17`? The main concern I have is that it may affect other repos, like `kitex`. Related issue: https://github.com/bytedance/gopkg/issues/163

Reply:
The go directive does not force the version of go compiler being used if no new API from higher version is involved. I have just tested this PR with go 1.16.15 and it works fine.

So changing the version or not shall have no impact on other repositories.

Reply:
Hmm, the required version upgrading maybe need another PR, if someone uses a different API.